utility/FIOaaS/README.md

```diff
@@ -0,0 +1,45 @@
+# ceph-fio-docker
+
+The end goal is to support fio for RBD and CephFs workloads.
+
+Currently RBD is supported.
+
+## RBD usage on rbd client machine
+
+[optional] Install docker in RHEL machines ( RBD client )
+
+```bash
+# yum config-manager --add-repo=https://download.docker.com/linux/centos/docker-ce.repo
+# yum install docker-ce -y
+# systemctl enable --now docker
+```
+
+Map rbd image as a block device
+
+```bash
+rbd map <pool>/<image>
+lsblk
+```
+
+Cleanup the block device 
+
+```bash
+mkfs.ext4 /dev/rbd<x>
```

Comment:
can we make the type of file system as user choice instead of hard code?

Reply:
All customisations are outside the scope aaS fio, any fs user can provide. Kindly watch the demo video on what is part of docker run command